resources/lang/fr/installer.php

```diff
@@ -45,7 +45,7 @@
     'permissions' => [
         'templateTitle' => 'Step 2 | Permissions',
         'title'         => 'Permissions',
-        'next'          => 'Configurer environnement',
+        'next'          => 'Configure environnement',
```

Comment:
"Configurer l'environnement" (Configure environment) is better i think

Reply:
Ici je ne sais pas s'il s'agit d'une option à cliquer, ou d'une information que le logiciel donne sur son travail en cours, dans le doute j'ai choisi Configure qui marche pour les deux options.
Faudrait réinstaller pour voir...

Reply:
Oui je me posais la question aussi mais je crois que c'est le titre de la page / d'un bouton